keras_contrib/applications/densenet.py

```diff
@@ -164,6 +166,114 @@ def DenseNet(depth=40, nb_dense_block=3, growth_rate=12, nb_filter=16,
     return model
 
 
+def DenseNetFCN(nb_dense_block=5, growth_rate=12, nb_filter=16, nb_layers_per_block=4,
```

Comment:
I didn't add it specifically bacause the original paper uses depth parameter to achieve this.

In FCN Densenet, there is no depth parameter. Instead you control depth with the two parameters nb_dense_blocks and nb_layers_per_block.